### ISSUE TITLE
Extension is not working with VS 2017 Professional

### ISSUE THREAD

**Issue by Maximus5** [Therena/ConEmuIntegration#10]
### Installed product versions
- Visual Studio: 2017 Professional (15.2.26430.16)
- This extension: 2.0.0.12 (built from sources)

### Description
Just rechecked the extension with VS 2017... Bunch of problems unfortunately

1. After the project is loaded (completely) right-click on the solution does not show any of ConEmu integration menu items. This is because `ConEmuWindowPackage::Initialize()` is called only when I open "Tools / Options" dialog and select "ConEmu integration" in options. I have already configured it, but I need to open options each time VS starts. I'm not familiar with VSIX to understand what is going wrong.
2. Even than, when extension is initialized (not completely?), when I choose "Open folder in ConEmu" from context menu, VS catch the exception

```
System.NullReferenceException occurred
  HResult=0x80004003
  Message=Object reference not set to an instance of an object.
  Source=ConEmuIntegration
  StackTrace:
   at ConEmuIntegration.SolutionExplorer.ExecuteInConEmu.ExecuteGuiMacro(String macro) in C:\....\Projects\conemu-helpers\ConEmuIntegration\ConEmuIntegration\SolutionExplorer\ExecuteInConEmu.cs:line 158
   at ConEmuIntegration.SolutionExplorer.OpenInConEmu.FolderOfProjectItem(SelectedItem selectedItem) in C:\....\Projects\conemu-helpers\ConEmuIntegration\ConEmuIntegration\SolutionExplorer\OpenInConEmu.cs:line 94
   at ConEmuIntegration.SolutionExplorer.OpenInConEmu.Open() in C:\....\Projects\conemu-helpers\ConEmuIntegration\ConEmuIntegration\SolutionExplorer\OpenInConEmu.cs:line 61
   at ConEmuIntegration.SolutionExplorer.OpenConEmuHere.MenuItemCallback(Object sender, EventArgs e) in C:\....\Projects\conemu-helpers\ConEmuIntegration\ConEmuIntegration\SolutionExplorer\OpenConEmuHere.cs:line 70
   at System.ComponentModel.Design.MenuCommand.Invoke()
   at System.ComponentModel.Design.MenuCommand.Invoke(Object arg)
   at Microsoft.VisualStudio.Shell.OleMenuCommandService.Microsoft.VisualStudio.OLE.Interop.IOleCommandTarget.Exec(Guid& commandGroup, UInt32 nCmdId, UInt32 nCmdExcept, IntPtr pIn, IntPtr vOut)
   at Microsoft.VisualStudio.Shell.Package.Microsoft.VisualStudio.OLE.Interop.IOleCommandTarget.Exec(Guid& guidGroup, UInt32 nCmdId, UInt32 nCmdExcept, IntPtr pIn, IntPtr vOut)
```

3, When I try "Execute in ConEmu" on the project root, I get error "Could not find file 'C:\....\conemu-helpers\ConEmuIntegration\ConEmuIntegration\bin\Debug\$(TargetPath)'." I doubt the `$(TargetPath)` should be there. Exception details below:

```
System.IO.FileNotFoundException occurred
  HResult=0x80070002
  Message=Could not find file 'C:\....\Projects\conemu-helpers\ConEmuIntegration\ConEmuIntegration\bin\Debug\$(TargetPath)'.
  Source=mscorlib
  StackTrace:
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.GetAttributes(String path)
   at ConEmuIntegration.SolutionExplorer.ExecuteInConEmu.RunExecutable(Project project) in C:\....\Projects\conemu-helpers\ConEmuIntegration\ConEmuIntegration\SolutionExplorer\ExecuteInConEmu.cs:line 130
   at ConEmuIntegration.SolutionExplorer.ExecuteInConEmu.MenuItemCallback(Object sender, EventArgs e) in C:\....\Projects\conemu-helpers\ConEmuIntegration\ConEmuIntegration\SolutionExplorer\ExecuteInConEmu.cs:line 115
   at Microsoft.VisualStudio.Shell.OleMenuCommand.Invoke(Object inArg, IntPtr outArg, OLECMDEXECOPT options)
   at Microsoft.VisualStudio.Shell.OleMenuCommandService.Microsoft.VisualStudio.OLE.Interop.IOleCommandTarget.Exec(Guid& commandGroup, UInt32 nCmdId, UInt32 nCmdExcept, IntPtr pIn, IntPtr vOut)
   at Microsoft.VisualStudio.Shell.Package.Microsoft.VisualStudio.OLE.Interop.IOleCommandTarget.Exec(Guid& guidGroup, UInt32 nCmdId, UInt32 nCmdExcept, IntPtr pIn, IntPtr vOut)
```

4. When I try "Open output path" in ConEmu, I get an error "Could not find file 'C:\....\Documents\Visual Studio 2017\Projects\ConsoleApplication2\Debug\ConsoleApplication2.exe'." No macros here, but why it checks for executable? User expects to open the folder `Debug` and the folder actually exists.

**Comment by Therena** [comment]
Thanks for handing in these problems.

I already know that there are still problems with C/C++ projects.
I'm currently working on a solution and will include it in one of the next releases.

I will keep you up to date.

**Comment by Therena** [comment]
Issue 1, 2 and 3 I could reproduce and tried to fix them with the new release 2.0.0.13.
The fourth issue I couldn't reproduce on my computer.
I will see in the next days if I can see it happening.

Can you please check as well if everything is fixed for you and give me feedback?

**Comment by Maximus5** [comment]
1, 2, 3 are fixed indeed, thanks!

4 is reproduced. Just do "Clean" on C++ project and try "Open outpath in ConEmu". Minor problem actually, but the exception appears.

**Comment by Therena** [comment]
4 should be fixed now as well (since 2.0.0.14)
